web-console/5-components/order-from-catalog.md

```diff
@@ -1,46 +1,38 @@
 # Ordering Experience
 
-The catalog will include both builder images and items from the Catalog API.  This experience allows the user to remain in context, by using the overlay panel.
-
-##  Open Questions / To Be Determined
-- Terminology - Should the action button be Create for both builder images & items from the catalog?
-- Results Panel - finalize the content and associated links
-- High Fidelity Mocks
+The catalog will include builder images, templates and items from the Broker API.  This experience allows the user to remain in context, by using the overlay panel.
 
 ## Design
-- Upon clicking a service, the Overlay Panel Component will be used to generate a guided experience.  
+- Upon clicking on an item in the catalog, the Overlay Panel Component will be used to generate a guided experience.  
 - The guided overlay panel has 3 main areas: top panel, content panel & button panel
 
 ![template](img/guided-overlay.png)
 
 ### Top Panel
-- The main steps of the wizard are shown in the bar along the top of the screen.
-- User can always see all of the primary steps in the flow at all times.  
-- This panel “sticky”, and also includes the item icon and item name.
+- The Catalog Item Name is displayed as the title of the overlay panel
+- The main steps of the wizard are shown under the title
+- This panel “sticky”, thus if the content panel scrolls, the top panel will not
 ![template](img/top-panel.png)
```

Comment:
@serenamarie125 This still looks like an old image with the item name on the left instead of the top